### PR TITLE
Add initial Kubernetes support

### PR DESCRIPTION
**Pull request by cedric-lamalle**
Hi,
This is just a first step for Kubernetes Integration. Basic features are working.
See https://github.com/cedric-lamalle/jhipster-control-center-k8s-sample for more details.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>tech.jhipster.controlcenter</groupId>
     <artifactId>jhipster-control-center</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>0.1.0-SNAPSHOT</version>
```
Comment (pascalgrimaud) [inline pom.xml:8]: why do you change this?

Comment (cedric-lamalle) [reply pom.xml:8]: Because of a new feature, but I will change it back.

Comment (cedric-lamalle) [reply pom.xml]: Projection Version reverted!

**Comment by PierreBesson** [comment]
Thanks @cedric-lamalle! Let me do some testing before merging. I did do the same changes as you in my previous poc when I was testing ideas for the JHipster Control Center. I remember that bringing in the spring-cloud-kubernetes dependency did create some beans in the application that might be unwanted depending on the profile.

**Pull request merged**